test/quantization/test_naive_intNwo.py

```diff
@@ -0,0 +1,46 @@
+import torch
```

Comment:
maybe call this `test_mixed_precision.py` to match your prototype folder and for your future test cases as well?

Reply:
Renamed it.